CHANGELOG.md

```diff
@@ -21,6 +21,10 @@
 
 - Running `bat` without arguments fails ("output file is also an input"), see #1396
 
+## Packaging
+
+- `bat` is now available on snapstore with package name called `batcat`, see #1401 (@purveshpatel511)
+
```

Comment:
Could you please move that up to the "unreleased" section? Right now, it's in 0.17.1, which has already been released

Reply:
@sharkdp sure.